Here comes New features : 3 ways answers and not available case

BenaeSan opened the pull request:
# Not available
## Issue #20 
a checkbox has been added to uncheck all the others checkbox, now you can show that you're not available for the meeting.

# 3 ways answers
Now you can create an advance poll, that give you the possibility to maybe be here for the meeting.

![capture_1](https://cloud.githubusercontent.com/assets/2964484/22472927/5aaff486-e7d7-11e6-9025-34247cc69d31.png)

![capture_2](https://cloud.githubusercontent.com/assets/2964484/22472928/5ac0de4a-e7d7-11e6-90e5-4d7287c3d0a0.png)

BenaeSan committed:
ADD
	attribut pollType
	+ traitement
	(WIP Non fonctionnel)
CHORE

BenaeSan committed:
ADD
	multiple value into slot selection
	create avdance answser poll
I18N

Commit:
ADD
	multiple value into slot selection
	create avdance answser poll
I18N

Commit:
FIX
	Simple Poll
I18N

Commit:
FIX
	Simple Poll
I18N

Commit:
CHORE
	clean code
I18N

Commit:
ADD
	not available checkbox whom pass to no all answer

Commit:
ADD
	not available checkbox whom pass to no all answer Only for simple poll
	js to perform check on some change

Commit:
ADD
	not avaible for advanced poll

Commit:
ADD
	page with the days selection in a plain datepicker
	create days widthout slots
	Slot page + action

Commit:
ADD
	page with the days selection in a plain datepicker
	create days widthout slots
	Slot page + action
	Copy the first line values to others

Commit:
ADD
	select date page
	script in a external file
	script to select date and remove it

Commit:
ADD
	delete selected date

Commit:
FIX
	new Slot get current hour
	First columns width get readjusted

Commit:
CHORE
	clean code
I18N

Commit:
FIX:
	calendar not working due to double initilisation
CHORE
	clean code
I18N

Commit:
Change maniftest and readme

Commit:
CHORE
	remove copyright header

Commit:
FIX
	datepicker css impact on all site

Commit:
ADD
	on change multi answer, the background color of cell change color to green/yellow/pink

Commit:
ADD
	on change, change background color of the cell

Commit:
CHORE
	Clean code

Commit:
ADD
	update days possibilty

Commit:
ADD
	roundToQuarter function
CHORE
	del unused file

juho-jaakkola commented:
Thanks for making the pull request!

Could you please change the indentations back to hard tabs (with width of 4 spaces)? The white space changes caused by them make it difficult to see what exactly has been changed.

Commit:
ADD
	update poll
	update date
	delete slot

Commit:
CHORE
	Change space to tab

BenaeSan commented:
Yeah Sure, i've done and commited. Hope it that you want.

Commit:
Add breadcrumb
I18N

Commit:
ADD
	error if no days' selected
FIX
	synch timestamp

Commit:
FIX
	problem with time UTC comparation
CHORE
	format code

Commit:
ADD
	answer object

Commit:
ADD
	answer object
FIX
	Save the answer object
	number of columns in addSlot Page
	order of the answer response to show the undefined slot

Commit:
FIX
	precheck case

Commit:
FIX 
	creation error if there is no vote value
CHORE
	clean code

Commit:
FIX 
	get the right vote value to update
CHORE
	clean code

Commit:
FIX
	update date or days

Commit:
FIX
	date format

Commit:
ADD
	name in main form

Commit:
FIX
	Counter for slot

Commit:
ADD
	Number of Voters

Commit:
ADD
	inner grid "pagination" repeat info all the tenth line

Commit:
FIX
	get the previous polltype value on update
CHORE
	clean & comment code

Commit:
Merge branch 'master' into feature_modify_poll

# Conflicts:
#	actions/scheduling/answer.php
#	actions/scheduling/days.php
#	actions/scheduling/save.php
#	classes/ElggSchedulingPoll.php
#	classes/ElggSchedulingPollSlot.php
#	languages/fr.php
#	views/default/forms/scheduling/answer.php
#	views/default/forms/scheduling/days.php

Commit:
FIX
	probleme affichage calendrier dans la V2

BenaeSan committed:
CHORE
	Migrate page handler route
	move file from pages to resources
	change icon to font-awesome
FIX
	some undefined variable
I18N
	some forgotten label

BenaeSan committed:
CHORE
	clean code

BenaeSan committed:
CHORE
	clean code
FIX
	fix uninstanciate object in advance answers

BenaeSan committed:
FIX
	fix notification problem on create poll

Commit:
Merge branch 'master' into master

juho-jaakkola commented:
Apologies that it has taken so long to take a proper look at this. I will try to get to it soon.

BenaeSan commented:
No problem, really

Commit:
fix: affichage date

BenaeSan committed:
I18N => FIX typo
